content/en/docs/concepts/storage/volumes.md

```diff
@@ -1057,7 +1060,7 @@ spec:
       path: /logs
```

Comment:
I think this example may not be as confusing if we make the hostPath.path = /var/log/pods

So then we can say something like "On the host, the logs are written to /var/log/pods/env-image/hello.txt"

Reply:
Maybe pod name should change to something like "pod1" too.

Reply:
Done